packages/php-db-import-export/src/Backend/Snowflake/SnowflakeException.php

```diff
@@ -30,25 +30,20 @@ public static function covertException(Throwable $e): Throwable
             );
         }
 
-        if (preg_match('/ \'(.*)\' is not recognized/', $e->getMessage(), $output_array) === 1) {
+        // phpcs:ignore
+        $isNullInNotNullCol = preg_match('/NULL result in a non-nullable column/', $e->getMessage(), $output_array) === 1;
+        $isNotRecognized = preg_match('/ \'(.*)\' is not recognized/', $e->getMessage(), $output_array) === 1;
+        if ($isNullInNotNullCol || $isNotRecognized) {
             return new Exception(
                 'Load error: ' . $e->getMessage(),
                 Exception::VALUE_CONVERSION,
                 $e
             );
         }
 
-        $isOutOfRange = preg_match(
-                '/.*out of range/',
-                $e->getMessage(),
-                $output_array
-            ) === 1;
-
-        $isBiggerThenColumnSize = preg_match(
-                '/ \'(.*)\' cannot be inserted because it\'s bigger than column size/',
-                $e->getMessage(),
-                $output_array
-            ) === 1;
+        $isOutOfRange = preg_match('/.*out of range/', $e->getMessage(), $output_array) === 1;
+        // phpcs:ignore
+        $isBiggerThenColumnSize = preg_match('/ \'(.*)\' cannot be inserted because it\'s bigger than column size/', $e->getMessage(), $output_array) === 1;
```

Comment:
tady sem taky nic neměnil, ale phpcs padalo

Reply:
trailing comma?

Reply:
Je nová verze phpcs venku totiž

Reply:
Ne začala mu vadit indentace

Reply:
![image](https://github.com/keboola/storage-backend/assets/180148/2f029f9f-cd89-4772-aad9-8390663ac6b1)

Reply:
Jo, vidim. No však je špatně, ne?

Reply:
Je tam o jeden tab navíc

Reply:
doteď nebyla nedělal sem tam změny

Reply:
Nj, prostě phpcs někde v minor verzi fixnul nějaký bug :)

Reply:
Klidně bych to revertnul a jen ododsadil o jeden tab :)